rio_alpha/findnodata.py

```diff
@@ -122,23 +122,23 @@ def determine_nodata(src_path, user_nodata, discovery, debug, verbose):
 
     with rasterio.open(src_path, "r") as src:
         count = src.count
-        data = np.rollaxis(src.read(), 0, 3)
 
-    if user_nodata:
-        return user_nodata
+        if user_nodata:
+            return user_nodata
```

Comment:
@yoninachmany generally I try to avoid increasing the scope of a PR, but what would you think about not opening the dataset at all in the `user_nodata` case?